mlflow/genai/evaluation/utils.py

```diff
@@ -36,6 +36,10 @@ def _convert_to_legacy_eval_set(data: "EvaluationDatasetTypes") -> "pd.DataFrame
     be accepted by mlflow.evaluate().
     The expected schema can be found at:
     https://docs.databricks.com/aws/en/generative-ai/agent-evaluation/evaluation-schema
+
+    NB: The harness secretly support 'expectations' column as well. It accepts a dictionary of
```

Comment:
nit: Do they plan to document it somewhere? Can we add a TODO comment to update the comment here for a correct reference?